requirements.txt

```diff
@@ -1,9 +1,9 @@
 h5py~=2.10.0
 Keras-Applications~=1.0.8
 Keras-Preprocessing~=1.1.0
-numpy~=1.17.3
+numpy~=1.19.2
 scipy~=1.3.1
-tensorflow==2.0.0
+tensorflow~=2.4.1
 Wave~=0.0.2
 
```

Comment:
does it need librosa?

Reply:
sure do. I thought there would be an error if I missed a dependency...